docs/readme-zephyr.md

```diff
@@ -148,15 +145,14 @@ The argument to `-t` should be the desired key type.  See the
 
 The generated keypair above contains both the public and the private
 key.  It is necessary to extract the public key and insert it into the
-bootloader.  The keys live in `boot/zephyr/keys.c`, and can be
-extracted using imgtool:
+bootloader.  ``CONFIG_BOOT_SIGNATURE_KEY_FILE`` Kconfig option shall be used to
+provide the path to the key file so the build system can extracting
+the public key to form which is usable by the C compiler.
+The public key is generated to the `build/zephyr/autogen-pubkey.h` which is included
+by the `boot/zephyr/keys.c`.
 
-```
-    $ ./scripts/imgtool.py getpub -k mykey.pem
-```
-
-This will output the public key as a C array that can be dropped
-directly into the `keys.c` file.
+Currently zephyr-rtos port limits its support to one keypair at the time,
+despite MCUboot's key managment infrastructure allows multiple keypairs.
```

Comment:
```suggestion
Currently, the Zephyr RTOS port limits its support to one keypair at the time,
even if MCUboot's key management infrastructure supports multiple keypairs.
```

Reply:
Not `even if` because MCUboot's infrastructure does support multiple key.